software/samba/en.md

```diff
@@ -138,9 +138,13 @@ sudo eopkg install kdenetwork-filesharing
 sudo eopkg install caja-extensions
 ```
 
-In order to load the newly installed file manager plugin, the user will need to log out of the current desktop session and log back in to a new desktop session, at which point the plugin in question will be ready for use.
+In order to load the newly installed file manager plugin, the user will need to log out of the current desktop session and log back in to a new desktop session, at which point the plugin in question will be ready for use.  To enable shares from any drive and folder on your System you will need add the following parameter to /etc/samba/smb.conf:
+```
```

Comment:
New empty line above ``` please :) Otherwise LGTM.

Reply:
Just changed it.  Still new to GitHub, so I hope I did it right.